modular_zubbers/code/modules/storyteller/storytellers/tellers/_storyteller.dm

```diff
@@ -50,6 +50,12 @@
 	if(disable_distribution)
 		return
 	add_points(delta_time)
+	if(EMERGENCY_AT_LEAST_DOCKED)
+		//Don't generate any new events if the shuttle is docked with the station (or in transit towards central command.
+		return
+	if( (SSshuttle.emergency_no_recall && !SSshuttle.admin_emergency_no_recall) && EMERGENCY_IDLE_OR_RECALLED)
+		//Don't generate any new events if the shuttle is in transit in a non-admin no-recall state.
+		return
```

Comment:
We can probably do this earlier in `modular_zubbers/code/modules/storyteller/gamemode.dm#L196`

Reply:
It is intentionally still there so manually run events by admins can still trigger, in case they want to run an event while the shuttle is coming.

Reply:
As well as allowing storyteller to generate point in case the shuttle was a hoax.

Reply:
> It is intentionally still there so manually run events by admins can still trigger, in case they want to run an event while the shuttle is coming.

The issue is, all events selected start with a 2-3 minute queue so then the issue isn't solved since they can get selected, wait 3 minutes, and spawn upon shuttle docking

Reply:
Then I will make it so that events are cleared at that point.

Reply:
Go ahead and you get the TM then merge